### PR TITLE
Added EXAMPLE_07, opening provider logins in a popup

### PR DESCRIPTION
**Pull request by neokio**
## Summary

* Addition

## Goal

Extend EXAMPLE_06 to open in a pop-up.

## Description

Added "EXAMPLE_07", which opens all authentication activity in a pop-up window.
Useful for projects where "never redirect visitors away from the site" is a requirement.

### CODE REVIEW

**Comment by ApacheEx** [comment]
do not see example_07 folder in your commit, can you check? 👍

**Comment by neokio** [comment]
Sorry about that, I didn't notice that your `.gitignore` was ignoring `/examples`!
I'm sure it was there for good reason, hope it's ok that I remove the line.

File: examples/example_07/index.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <meta charset="UTF-8">
+    <title>Example 07</title>
+</head>
+<body>
+    <h1>Sign in</h1>
+
+    <ul>
+
+<?php
+    foreach ($hybridauth->getProviders() as $name) {
```
Comment (ApacheEx) [inline examples/example_07/index.php]: let's use:

```php
    <?php foreach ($hybridauth->getProviders() as $name) : ?>
        <?php if (!isset($adapters[$name])) : ?>
            <li>
                <a href="#" onclick="auth_popup('<?php print $name; ?>');">
                    Sign in with <strong><?php print $name; ?></strong>
                </a>
            </li>
        <?php endif; ?>
    <?php endforeach; ?>
```

File: examples/example_07/index.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                Sign in with '. $name .'
+            </a>
+        </li>';
+        }
+    }
+?>
+
+    </ul>
+
+<?php if ($adapters) { ?>
```
Comment (ApacheEx) [inline examples/example_07/index.php]: let's use:

```php
<?php if ($adapters) : ?>
    <h1>You are logged in:</h1>
    <ul>
        <?php foreach ($adapters as $name => $adapter) : ?>
            <li>
                <strong><?php print $adapter->getUserProfile()->displayName; ?></strong> from
                <i><?php print $name; ?></i>
                <span>(<a href="<?php print $config['callback'] . "?logout={$name}"; ?>">Log Out</a>)</span>
            </li>
        <?php endforeach; ?>
    </ul>
<?php endif; ?>
```

File: examples/example_07/index.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    </ul>
+<?php } ?>
+
+</body>
+</html>
+
+
+
+
+<a href="#" onclick="javascript:auth_popup('Google');">Sign in with Google</a>
```
Comment (ApacheEx) [inline examples/example_07/index.php]: this should be inside `body` and commented via `<!-- >`

File: examples/example_07/index.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+<a href="#" onclick="javascript:auth_popup('Facebook');">Sign in with Facebook</a>
+
+
+<script>
```
Comment (ApacheEx) [inline examples/example_07/index.php]: move it to `head` section

**Comment by ApacheEx** [comment]
Looks good for me, thank you for your contribution 💯

**Pull request merged**